onward/app/views/fragments/feedbackForm.scala.html

```diff
@@ -0,0 +1,30 @@
+@(category: String)(implicit request: RequestHeader)
+
+<div>
+
+    <form action="#" method="post">
+
+        <input type="hidden" name="category" value="@category" />
+
+        <input class="feedback__element" name="extra" type="hidden" value="none available"/>
+
+        <div class="feedback__element">
+            <label class="" for="test">Description</label>
+            <textarea name="body" class="feedback__entry feedback__input--wide feedback__textarea" placeholder="Help us out with a friendly description of your problem" required></textarea>
+        </div>
+        <div class="feedback__element">
+            <label class="" for="test">Name</label>
+            <input name="name" class="feedback__entry feedback__input--wide" id="test" type="text" value="" placeholder="Alex Smith" required>
+        </div>
+        <div class="feedback__element">
+            <label class="" for="test">Email</label>
+            <input name="user" class="feedback__entry feedback__input--wide" id="test" type="email" value="" placeholder="E.g. alexsmith@@example.com" required>
```

Comment:
Element `id`s must be unique on the page. The `for` field in the label should refer to the `id` in the input